src/main/java/com/slimebot/games/games/Wordchain.java

```diff
@@ -0,0 +1,210 @@
+package com.slimebot.games.games;
+
+import com.slimebot.games.Game;
+import com.slimebot.main.Main;
+import net.dv8tion.jda.api.EmbedBuilder;
+import net.dv8tion.jda.api.entities.MessageEmbed;
+import net.dv8tion.jda.api.events.interaction.component.ButtonInteractionEvent;
+import net.dv8tion.jda.api.events.message.MessageReceivedEvent;
+import net.dv8tion.jda.api.interactions.components.buttons.Button;
+import net.dv8tion.jda.api.requests.RestAction;
+
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.ScheduledFuture;
+import java.util.concurrent.TimeUnit;
+import java.util.stream.Collectors;
+
+public class Wordchain extends Game {
+    private String lastWord;
+    private List<String> words;
+    private int playerTurn;
+    private int round;
+    private ScheduledFuture<?> scheduledFuture;
+    private final short seconds;
+    private final short lives;
+    private Map<Long, Short> playerLives;
+
+    public Wordchain(long gameMaster, long channel, long guildId, short seconds, short lives) {
+        super(gameMaster, channel, guildId);
+        this.playerLives = new HashMap<>();
+        this.seconds = seconds;
+        this.lives = lives;
+        this.words = new ArrayList<>();
+
+        Main.jdaInstance.getGuildById(guildId).retrieveMemberById(gameMaster).queue(m -> {
+            getChannel().sendMessageEmbeds(
+                            new EmbedBuilder()
+                                    .setColor(Main.embedColor(String.valueOf(guildId)))
+                                    .setTitle(m.getEffectiveName() + " hat eine neues Wortketten Spiel erstellt!")
+                                    .setDescription("Um zu erfahren wie \"Wortkette\" funktioniert nutze ```/wordchain explanation```") // TODO
+                                    .addField("Spielleiter:in:", "<@"+gameMaster+">", true)
+                                    .addField(":timer: Timeout:", seconds + "s", true)
+                                    .addField(":x: Max Fehler:", lives + " Fehler", true)
+                                    .addField("Spieler:", player.stream().map(p -> ("<@"+ p + ">")).collect(Collectors.joining("\r\n")), true)
```

Comment:
I would add a protected method in the Game class `buildPlayerList` that returns the mentions of all players. It would look something like this: `players.stream.map(UserSnowflakre::getAsMention).collect(Collectos.joining("\n")`. Btw. I think `\n` is enough, you don't need `\r\n`.